code/config/lib/Loot/Config/HasLens.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE ConstraintKinds      #-}
+{-# LANGUAGE DataKinds            #-}
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE FlexibleInstances    #-}
+{-# LANGUAGE GADTs                #-}
+{-# LANGUAGE KindSignatures       #-}
+{-# LANGUAGE OverloadedLabels     #-}
+{-# LANGUAGE PolyKinds            #-}
+{-# LANGUAGE StandaloneDeriving   #-}
+{-# LANGUAGE TemplateHaskell      #-}
+{-# LANGUAGE TypeFamilies         #-}
+{-# LANGUAGE TypeOperators        #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- | This module provides orphan instance HasLens for config.
+module Loot.Config.HasLens  where
+
+import Data.Vinyl (Label (..))
+import GHC.TypeLits (ErrorMessage (Text), Symbol, TypeError)
+
+import Loot.Base.HasLens (HasLens (..))
+import Loot.Config.Record
+
+----------------------------------------------------------------------------
+-- Constraints
+----------------------------------------------------------------------------
+
+-- Because there is no publicly availble type family for list concatenation.
+type family (++) (as :: [k]) (bs :: [k]) :: [k] where
```

Comment:
Do you really-really need `PolyKinds` here, not plain `[*]`?
This extension delivers fair amount of pain, both in compiler messages and typeclass instances writing if you happened to define typeclass with type variables of unusual kinds

Reply:
Yes, it's required, I operate with lists of `ItemKind`